examples/mnist/README.md

```diff
@@ -0,0 +1,69 @@
+# MNIST
+
+
+We show an example of building a cnn network trained on MNIST dataset.
+
+This example includes the following steps:
+
+- Loading MNIST dataset and do some very basic preprocessing
+- Building a CNN network using tensorflow keras modules
+- Sanity Check: testing the (untrained) model prediction inference on local environment.
+- Setting up Leap env
+- Integrate the model and dataset into the TL.
+
```

Comment:
I understand what you mean about this, but I am not sure that a user could understand what you mean.
Either we are explaining each of these steps, or we should remove this, or replace it with a more relevant points.